src/pages/LoginForm.tsx

```diff
@@ -38,6 +38,7 @@ export var currentUser: User | undefined = {
   ready: false,
   warningOne: "",
   warningTwo: "",
+  isSuspicious: false,
```

Comment:
What happens if a user starts an exam, gets flagged as suspicious, logs out and back in. Does the suspicious flag persist?

Reply:
Student can not log back in until exam is done